core/src/main/java/hudson/model/UsageStatistics.java

```diff
@@ -190,8 +196,10 @@ public String getStatData() throws IOException {
             }
 
             return new String(Base64.getEncoder().encode(baos.toByteArray()));
-        } catch (GeneralSecurityException e) {
-            throw new Error(e); // impossible
```

Comment:
> impossible

😕❓

Reply:
This is a 13yrs old line of code, coming from the subversion and the details on the commit is limited. 
I'm not sure if the exception was not possible or if the instance was supposed to survive past a `GeneralSecurityException`. 

How would you want this to be handled?